docs/quickstarts/excel-quickstart-angular.md

```diff
@@ -69,7 +69,6 @@ Congratulations, you've successfully created an Excel task pane add-in using Ang
 ## See also
 
 * [Office Add-ins platform overview](../overview/office-add-ins.md)
-* [Building Office Add-ins](../overview/office-add-ins-fundamentals.md)
 * [Develop Office Add-ins](../develop/develop-overview.md)
```

Comment:
Minor tweak to this article title, to keep it consistent with the other references to the title in this PR. 
```suggestion
* [Developing Office Add-ins](../develop/develop-overview.md)
```